types/index.d.ts

```diff
@@ -54,7 +53,7 @@ export function cleanup(): void
 export type FireFunction = (element: Document | Element | Window, event: Event) => Promise<boolean>;
 
 export type FireObject = {
-  [K in EventType]: (element: Document | Element | Window, options?: {}) => Promise<boolean>;
+  [K in EventType]: (element: Document | Element | Window, options?: Record<string, uknown>) => Promise<boolean>;
```

Comment:
Oops. `uknown` typo.

And while `string` is the usual type for the key, if we want to be pedantically equivalent to `{}`, that should be `string|number|Symbol`, I think? (although this is for options, which will only have string keys, so yeah, that can probably stay as `string` only.

Reply:
Decided to revert back to `{}` and silence the lint warning, because `{}` is what dom-testing-library uses